pkg/providers/vsphere/virtualmachine/snapshot.go

```diff
@@ -32,9 +33,10 @@ type SnapshotArgs struct {
 
 // Snapshot related errors.
 var (
-	ErrNoSnapshots       = errors.New("no snapshots for this VM")
-	ErrSnapshotNotFound  = errors.New("snapshot not found")
-	ErrMultipleSnapshots = errors.New("multiple snapshots found")
+	ErrSnapshotNotFound        = errors.New("snapshot not found")
+	errNoSnapshots             = errors.New("no snapshots for this VM")
+	errMultipleSnapshots       = errors.New("multiple snapshots found")
+	errSnapshotMoRefNilOrEmpty = errors.New("snapshotMoRef is nil or empty")
```

Comment:
Why make this a non-exportable sentinel error?

Reply:
Yeah this is only used once, let me change that